realm/realm-library/src/main/java/io/realm/internal/SharedRealm.java

```diff
@@ -175,6 +192,16 @@ public boolean isInTransaction() {
         return nativeIsInTransaction(nativePtr);
     }
 
+    public void setSchemaVersion(long schemaVersion) {
+        // FIXME migrate to ObjectStore
```

Comment:
Making it public in the Object Store would make be feel much safer. This is a table that _must not_ be different than other platforms.